experiments/configs/dpd_image_classifier.yaml

```diff
@@ -0,0 +1,17 @@
+defaults:
+  - workspace: M365Research-PPML-EUS
+  - _self_
+
+shared_training_parameters:
+  total_train_batch_size: 512
+  target_epsilon: 8.0
+  num_train_epochs: 10.0
+  learning_rate: 0.5
+  max_physical_batch_size: 512
+  per_sample_max_grad_norm: 2.6
+  delta: 1e-5
+  lr_scheduler_gamma: 0.96
+
+game_config:
+  seed: 1920
+  num_games: 2
```

Comment:
Should we remove this file / add it to gitignore?

Reply:
I think it can be useful to have a working set of hyper-parameters. It's in the experiment subdir so I hope it's clear that it's meant to be an example. I have removed the details of the workspace as they won't be helpful for anyone without access.